virttest/cartesian_config.py

```diff
@@ -1959,7 +1959,7 @@ def get_dicts(self, node=None, ctx=[], content=[], shortname=[], dep=[]):
 
     # Make name for test. Case: two dics were merged
     def mk_name(self, n1, n2):
-        common_prefix = n1[:[x[0] == x[1] for x in zip(n1, n2)].index(0)]
+        common_prefix = n1[:[x[0] == x[1] for x in list(zip(n1, n2)].index(0))]
```

Comment:
Travis CI failed due to the wrong ')', it should follow 'n2' instead of 'index(0)'